docs/api.md

```diff
@@ -273,7 +279,7 @@ for `code` and `errno` are:
   This email can not currently be used to login
 * `code: 400, errno: 150`:
   Can not resend email code to an email that does not belong to this account
-* `code: 422, errno: 151`:
+* `code: 500, errno: 151`:
```

Comment:
It's possible this is because of some kind of non-deterministic ordering in the parse tree that the doc script runs against. `errno` 151 can be either 422 or a 500, which may have been a poor decisions on my part, and the doc script is very naive in its efforts to pull out error codes.